app/src/main/java/org/p2p/wallet/restore/ui/derivable/DerivableAccountsAdapter.kt

```diff
@@ -27,22 +31,22 @@ class DerivableAccountsAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>()
     override fun getItemCount(): Int = data.size
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder =
-        ViewHolder(parent)
+        ViewHolder(
+            binding = ItemTokenSimpleBinding.inflate(
```

Comment:
binding = parent.inflateViewBinding(attachToParent = false)

Reply:
but we extract creating binding inside the constructor, like org.p2p.wallet.home.ui.new.adapter.CategoryTitleViewHolder

Reply:
yeah, that's a great example, it's much better than having separate constructor to bind view as it was before